keras_nlp/layers/modeling/f_net_encoder.py

```diff
@@ -93,17 +96,19 @@ def __init__(
         self.kernel_initializer = keras.initializers.get(kernel_initializer)
         self.bias_initializer = keras.initializers.get(bias_initializer)
 
-    def build(self, input_shape):
+    def build(self, inputs_shape):
```

Comment:
Why the rename here?

Reply:
no longer strictly necessary, at one point this was with keras-core.

but the general rule is that build arguments should match call arguments + `_shape`. in the one argument case, any name fine.